test/functionality/abstract-interpretation/data-frame/data-frame.ts

```diff
@@ -32,56 +32,74 @@ export const DataFrameTestOverapproximation = {
 	rows:     DomainMatchingType.Overapproximation
 };
 
+interface CriterionTestEntry {
+	criterion:  SingleSlicingCriterion,
+	value:      DataFrameDomain,
+	node:       RSymbol<object>,
```

Comment:
Keeping the additional info as `object` appears to be unreasonably vague. Why not require the `ParentInformation` + your absint requirements?